frontend/tsconfig.json

```diff
@@ -19,7 +19,8 @@
     "target": "esnext",
     "types": ["jest-playwright-preset", "expect-playwright", "jest"],
     "forceConsistentCasingInFileNames": true,
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "incremental": true
```

Comment:
this is enabled by nextjs, which speeds up TS compilation

https://www.typescriptlang.org/tsconfig#incremental